packages/wayland.rb

```diff
@@ -39,5 +39,7 @@ def self.build
 
   def self.install
     system "DESTDIR=#{CREW_DEST_DIR} ninja -C builddir install"
+    FileUtils.mkdir_p("#{CREW_DEST_PREFIX}/etc/env.d/")
+    File.write("#{CREW_DEST_PREFIX}/etc/env.d/exo_base", @env)
```

Comment:
Would `wayland` be a better name than `exo_base`?

Reply:
Changed to `wayland`